examples/plotting/time_distance_slice_plot.py

```diff
@@ -0,0 +1,104 @@
+"""
+============================================
+Time-Distance Slice Plot of Sequence of Maps
+============================================
+
+This example showcases how you can use :func:`sunpy.map.pixelate_coord_path` and
+:func:`sunpy.map.sample_at_coords()` on a sequence of images to extract a time-distance
+slice and identify features in the solar atmosphere.
+"""
+import matplotlib.pyplot as plt
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+from astropy.visualization import ImageNormalize, SqrtStretch
+
+import sunpy.map
+from sunpy.coordinates import propagate_with_solar_surface
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+
+###############################################################################
+# First, we will import a sequence of images from AIA 171 angstrom separated by 1 hour
+# and create a sequence of maps.
+
+query = Fido.search(a.Time('2018-05-30 02:00:00', '2018-05-30 10:00:00'),
+                    a.Instrument.aia,
+                    a.Wavelength(171*u.angstrom),
+                    a.Sample(1*u.hr))
+print(query)
+files = Fido.fetch(query)
+aia_sequence = sunpy.map.Map(files, sequence=True)
```

Comment:
You should plot the first one in sequence and plot the object you want to slice on top of it.